CCS/3. Tutorials/8. 視覺化呈現資料分佈－用 TensorFlow 實作 Linear-Regression.md

```diff
@@ -1 +1,167 @@
+---
+title: 視覺化呈現資料分佈 Linear R
+tags: HowTo, CCS, Interactive, TW
+GA: UA-155999456-1
+---
 
+{%hackmd @docsharedstyle/default %}
+
+# HowTo: Visualize data distribution-implement Linear-Regression with TensorFlow
+
+
+## 1. Using Jupyter Notebook (Python)
+
+### Step 1. Sign in TWCC
+
+- If you don’t have an account yet, please refer to [Register for an TWCC Account](https://www.twcc.ai/doc?page=register_account)
+
+### Step 2.  Create a Interactive Container
+
+- Please refer to [Interactive Container](https://www.twcc.ai/doc?page=container#建立開發型容器) to create a Interactive Container.
+- Please select TensorFlow for the image file type (For the image file, please select the version before 19.08 (`not including 19.08`), and the version of Python 2)
```

Comment:
[SUG]

1. image file type > image type
2. the image file > image
3. the version > a version
4. , and the version of Python 2 > with Python 2